pyanaconda/modules/payloads/payload/dnf/installation.py

```diff
@@ -90,6 +95,51 @@ def _install_macros(self, macros):
             rpm.addMacro(name, value)
 
 
+class ResolvePackagesTask(CheckPackagesSelectionTask):
```

Comment:
I can't help myself but this doesn't look logically correct. I don't think that Resolve should extend Check class.

However, it is definitely not something which should block this PR.

Reply:
It would help to have some more context. I just assumed there will be more classes extending that...